hack/manifest-gen/README.md

```diff
@@ -0,0 +1,44 @@
+ECK Manifest Generator
+======================
+
+Utility for generating ECK installation manifests.
+
+Quickstart
+----------
+
+NOTE: Requires Docker and Make to be installed on your computer.
+
+Cluster-wide installation with access to all namespaces:
+
+```shell
+make docker-gen-global | kubectl apply -f -
+```
+
+Installation restricted to a single namespace named "myns":
+
+```shell
+make docker-gen-restricted RESTRICTED_NS=myns | kubectl apply -f -
+```
+
+
+Advanced Usage
+--------------
+
+Generated manifests can be customized further to suit your needs. This requires having a working Docker image of the manifest generator:
+
+```shell
+make docker-build DOCKER_IMAGE=manifest-gen
+```
+
+Run the following command to view documentation and examples of customizing manifests:
+
+```shell
+docker run manifest-gen generate --help
+```
+
```

Comment:
I thought we were not officially supporting Helm -- hence the need for this in the first place. Otherwise we could just publish an official Helm chart, can't we?

Reply:
Yeah you're right.